src/components/atoms/Logo/index.tsx

```diff
@@ -1,16 +1,10 @@
 import * as React from 'react'
-import { StyleLogoContainer, StyleLogo } from "./styles"
+import { StyleLogoContainer, StyleLogo, StyleLinkName } from "./styles"
 
-interface LogoLinkInterface {
-    logoUrl: string,
-}
-
-const Logo: React.FC<LogoLinkInterface> = (
-    props: LogoLinkInterface
-) => {
+const Logo = () => {
```

Comment:
This is correct. But, since the website needs to be SEO friendly you need to refactor this component. 
Basically, the img logo will be inside an `h1`tag

You can do 
```
<h1>
<Link>
// It's all up to you whether you want to lazy load this img
<img/>
</Link>
</h1>
```

or

```
<h1>
<Link>
// you will receive the svg from props. Since it's an svg (HTML) you don't need to lazy load anything. 
<svg>
</Link>
</h1>
```

I'll probably do the second option. 


https://stackoverflow.com/questions/665037/replacing-h1-text-with-a-logo-image-best-method-for-seo-and-accessibility

Reply:
You can see Mipon's desktop version as a reference.